client/lib/inflight/index.js

```diff
@@ -1,16 +1,24 @@
-var inflight = {};
+const inflight = new Set();
 
 // Utility methods to track inflight async requests
-module.exports = {
+export default {
```

Comment:
if you're going to go this road, I'd rather use named exports and fix up all the users of `lib/inflight` then export an object with props.